graphql_api/schema/custom/openquake_hazard_solution.py

```diff
@@ -40,7 +40,10 @@ class Meta:
 
     created = graphene.DateTime(description="When it was created (UTZ)")
     config = graphene.Field(
-        OpenquakeHazardConfig, description="the template configuration used to produce this solution"
+        OpenquakeHazardConfig,
+        description="the template configuration used to produce this solution",
+        required=False,
+        deprecation_reason="We no longer use this field",
     )
```

Comment:
We should probably deprecate modified_config as well

Reply:
I'll do it here. I've started on runzi, and realised that they are so tightly coupled.